samples/encoder/main.cpp

```diff
@@ -0,0 +1,66 @@
+#include <EVT/dev/LCD.hpp>
+#include <EVT/dev/encoder.hpp>
+#include <EVT/io/UART.hpp>
+#include <EVT/io/pin.hpp>
+#include <EVT/manager.hpp>
+#include <EVT/utils/time.hpp>
+#include <cstdio>
+
+namespace IO = EVT::core::IO;
+namespace DEV = EVT::core::DEV;
+namespace time = EVT::core::time;
+
+constexpr IO::Pin A_PIN = IO::Pin::PA_8;
+constexpr IO::Pin B_PIN = IO::Pin::PA_9;
+
+//LCD stuff
```

Comment:
The encoder sample should not have anything related to the charge controller LCD in it

Reply:
yeah accidental leftover from debugging, fixed that!